config/sbnd/flashmatch_250408.cfg

```
@@ -0,0 +1,367 @@
+
+FlashMatchManager: {
+  Verbosity: 2
+  AllowReuseFlash: true
+  StoreFullResult: false
+  FlashFilterAlgo: "BeamWindowFlashFilter"
+  TPCFilterAlgo:   "NPtFilter"
+  ProhibitAlgo:    "" # "TimeCompatMatch"
+  HypothesisAlgo:  "SemiAnalyticalModel" #"ChargeAnalytical" #"SemiAnalyticalModel" #"PhotonLibHypothesis" # #
+  MatchAlgo:       "QLLMatch"
+  CustomAlgo:      ["LightPath"] #,"MCQCluster"] # Keep lightpath,though it's not used now
+  MatchSelectionAlgo: "SelectionGreedy" # Not used in SBND
+  ChannelTypes: [] #doesn't do anything
+  UncoatedOpDets: [] #doesn't do anything
+  ChannelsToUse: [6,7,8,9,10,11,12,13,14,15,16,17,36,37,38,39,40,41,60,61,62,63,64,65,66,67,68,69,70,71,84,85
```

```suggestion
  ChannelToUse: [6,7,8,9,10,11,12,13,14,15,16,17,36,37,38,39,40,41,60,61,62,63,64,65,66,67,68,69,70,71,84,85
```
For some reason, this is the name chosen. `ChannelToUse`